siriuspy/siriuspy/devices/device.py

```diff
@@ -91,14 +91,15 @@ def values(self):
 
     def __getitem__(self, propty):
         """Return value of property."""
-        pvobj = self._pvs[propty]
-        value = pvobj.get()
-        return value
+        pvobj = self._pvs.get(propty)
```

Comment:
pyling `inconsistent-return-statement`

Reply:
I don't understand. I don't get these errors.
What is it complaining about?

Reply:
done!